src/Microsoft.ML.OnnxTransform/Microsoft.ML.OnnxTransform.csproj

```diff
@@ -4,12 +4,13 @@
     <TargetFramework>netstandard2.0</TargetFramework>
     <IncludeInPackage>Microsoft.ML.OnnxTransform</IncludeInPackage>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <ResolveAssemblyWarnOrErrorOnTargetArchitectureMismatch>None</ResolveAssemblyWarnOrErrorOnTargetArchitectureMismatch>
```

Comment:
Why is this necessary?

If it is because `Microsoft.ML.OnnxRuntime` builds their .NET assembly as x64, we should get them to stop that.  In general, IL should be built for `AnyCPU`. It's the C++ assembly that will be built for x64 or x86 (or arm). And those can be added later, without changing the .NET assembly.

Reply:
I was initially getting a persistent long warning about differences between DLL  architectures (onnxruntime.dll=AMD64 versus the project= MSIL), so was using this tag to suppress the warning.  However, the warning is not present  in the latest onnxruntime nuget -- so we don't need to suppress it any more. I'll remove this tag.